### ISSUE TITLE
Supply data in JOSM Maps directly

### ISSUE THREAD

**Issue by stoecker** [maxerickson/NAIP_Remote#1]
Hello,

due to a post in JOSM dev list I've seen your project.

https://lists.openstreetmap.org/pipermail/josm-dev/2017-February/007788.html

While it's a fine idea to manage a link list, JOSM has already a much better system to provide imagery.

At
https://josm.openstreetmap.de/wiki/Maps
or more specific
https://josm.openstreetmap.de/wiki/Maps/USA

XML formatted entries can be added describing the imagery link as well as additional information. Everything entered there is automatically provides in JOSM. The wiki is freely editable (as long as the license of the entries is ok for OSM).

When proper shapes are added in the XML, then imagery is also automatically shown in menu when working in the appropriate area.

For such large number of data you could also make an own page like
https://josm.openstreetmap.de/wiki/Maps/USA-NAIP
if you want.

**Comment by maxerickson** [comment]
Yeah I realize the advantages and I've been looking at doing that. I put this together mostly for my own use and it didn't make any sense at that point not to share it.

I mostly didn't want to take the time to generate coverage areas, I hadn't considered that the links could be added to the JOSM wiki without them. But it makes sense to add them and then work on adding the areas.

Thanks for reaching out.
